<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ПРМ-300 Ethernet</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="minimumSize">
         <size>
          <width>88</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="numDigits">
         <number>4</number>
        </property>
        <property name="mode">
         <enum>QLCDNumber::Dec</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dial">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>7000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="sliderPosition">
         <number>20</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>317</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Настройка приемника</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="spinBox_Freq">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="suffix">
              <string> МГц</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>20</number>
             </property>
             <property name="maximum">
              <number>7000</number>
             </property>
             <property name="singleStep">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_At1">
             <item>
              <property name="text">
               <string>0 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10 дБ</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_At2">
             <property name="maxVisibleItems">
              <number>16</number>
             </property>
             <item>
              <property name="text">
               <string>0 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>11 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>14 дБ</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15 дБ</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Fltr">
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>140/70 МГц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>140/22 МГц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>140/20 МГц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>150/80 МГц</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_set">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Установить</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_get">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Запросить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="minimumSize">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>101</width>
            <height>20</height>
           </size>
          </property>
          <property name="inputMask">
           <string>000.000.000.000; </string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_connect">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Подключение к приемнику</string>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Подключиться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
